package.json

```diff
@@ -78,6 +78,7 @@
     "fast-safe-stringify": "^2.0.4",
     "http-headers": "^3.0.2",
     "is-native": "^1.0.1",
+    "ms": "^2.1.1",
```

Comment:
The ms package uses a [much looser time suffix parser](https://www.npmjs.com/package/ms#examples) than what we have decided to support. This theoretically means that a user can write `2.5 hrs` instead of `150m`. This will work fine with the Node.js agent, but if they try to use the same value for the RUM JavaScript agent or the Java agent, it might fail.

If I recall correctly from when we originally discussed this, we wanted to only allow a strict subset of suffixes. Is that still the case?

cc @elastic/apm-agent-devs

Reply:
Yes, the intention was to be rather strict in order to align across agents to support the case when different agents read from the same environment variable or, going forward, a centralized configuration.

Reply:
Thanks @felixbarny 

@Qard Instead of reinventing the well, we could continue to use the `ms` package, but add a regex to validate the string before passing it along to the `ms` package. That would be a quick and easy solution. What do you think about that?

Reply:
If we want to limit it that much, it's probably better to just skip ms altogether. It's a fairly basic conversation map when there are only three suffixes. 🤔

Reply:
I've replaced ms with a simple `toSeconds` function that uses the regex with only `s`, `m`, and `ms` support.